README.md

```diff
@@ -12,8 +12,51 @@ Please, post an issue on GitHub and **provide a plunker** with your question.
 ```javascript
 npm install ng2-eonasdan-datetimepicker --save
 ```
-
-2) Add the A2Edatetimepicker module import in your module.:
+2) Include all required scripts if it doesn't included yet in your project:
+ - jquery.js, 
+ - bootstrap (.js & .css),
+ - eonasdan bootsrap datetimepicker (.js & .css),
+ 
+ - moment.js.
+
+   (nb: you don't need to do another npm install, the previous command already did it)
+    #### Angular cli:
+    an example by using angular cli, you can add theses libs in scripts and styles keys in  **.angular-cli.json**
+
+
+    ```json 
+    "styles": [
+        "styles.css",
+        "../node_modules/bootstrap/dist/css/bootstrap.min.css",
+        "../node_modules/eonasdan-bootstrap-datetimepicker/build/css/bootstrap-datetimepicker.min.css"
+      ],
+      "scripts": [
+        "../node_modules/jquery/dist/jquery.min.js",
+        "../node_modules/bootstrap/dist/js/bootstrap.min.js",
+        "../node_modules/moment/min/moment.min.js",
+        "../node_modules/eonasdan-bootstrap-datetimepicker/src/js/bootstrap-datetimepicker.js"
+      ],
+    ```
+    #### Webpack:
+    another example by using [angular webpack described here](https://angular.io/guide/webpack) : import the librairies  in the **vendor.ts** file
```

Comment:
Are you sure about this config? 
Have you tested it yourself?

Reply:
Yes, I've tested it before the PR, if you want I can push with examples?

Reply:
yes, what do you suggest to structure? or having a gh-pages also with a live demo page is useful.

Reply:
just add it in `/examples/webpack` and `/examples/angular-cli` subfolders

Reply:
also ignore it in package.json.

Reply:
what do you prefer for ignoring folder? using .npmignore or by specifiyng all including files in files?

Reply:
either is fine